specs/src/pages/widgets/frequently-bought-together.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: ../../layouts/WidgetLayout.astro
+title: FrequentlyBoughtTogether
+type: widget
+html: |
+  <div class="ais-FrequentlyBoughtTogether">
+    <h3 class="ais-FrequentlyBoughtTogether-title">Frequently bought together</h3>
+    <div class="ais-FrequentlyBoughtTogether-container">
+      <ol class="ais-FrequentlyBoughtTogether-list">
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 5477500: Amazon - Fire TV Stick with Alexa Voice Remote - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 5477500: Amazon - Fire TV Stick with Alexa Voice Remote - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 5477500: Amazon - Fire TV Stick with Alexa Voice Remote - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+      </ol>
+    </div>
+  </div>
+classes:
+  - name: .ais-FrequentlyBoughtTogether
+    description: the root div of the widget
+  - name: .ais-FrequentlyBoughtTogether-title
+    description: the title heading of the widget
+  - name: .ais-FrequentlyBoughtTogether-container
+    description: the container for the list of results
+  - name: .ais-FrequentlyBoughtTogether-list
+    description: the list of results
+  - name: .ais-FrequentlyBoughtTogether-item
+    description: the hit list item
+options:
+  - name: transformItems
+    description: Function which receives the items, which will be called before displaying them. Should return a new array with the same shape as the original array. Useful for mapping over the items to transform, remove or reorder them
+  - name: escapeHTML
```

Comment:
This isn't exposed in FBT.

Reply:
This is forwarded to the connector and is handled on the frontend side (the param is not sent to Algolia).

Reply:
I've removed it to reflect its removal in the connector.